src/main/java/com/habibank/controller/HabibankController.java

```diff
@@ -90,14 +88,15 @@ private String index() {
 
 
 
-	
-//	@PostMapping("api/employees")
-//	private ResponseEntity<?> createEmp(@RequestBody Account e) {
-//		Account createdAccount = new Account(
-//				e.getFullName(),e.getDep(),e.getJobTitle(), e.getYearlySalary());
-//		this.acctRepo.saveAndFlush(createdAccount);
-//		URI location = ServletUriComponentsBuilder.fromCurrentRequest().path("/{id}").buildAndExpand(createdAccount.getEmpID()).toUri();
-//		return ResponseEntity.created(location).build();
-//	}
+//create customer	
+	@PostMapping("api/customers")
+	private ResponseEntity<?> createEmp(@RequestBody Customer c) {
+		Customer createdCustomer = new Customer(
+				c.getUserName(),c.getFullName(),c.getEmail(),
+				c.getAddress(),c.getPhoneNumber());
+		this.custRepo.saveAndFlush(createdCustomer);
+		URI location = ServletUriComponentsBuilder.fromCurrentRequest().path("/{id}").buildAndExpand(createdCustomer.getCustomerID()).toUri();
+		return ResponseEntity.created(location).build();
+	}
```

Comment:
This all looks good to me. Good job, I would say just _change createEmp to createCust_ on naming.  I was just stuck and thinking whether we should include the password in the post creation for the validation side of the API React side and Spring boot side.  This does look good for testing

Reply:
true thats a vestige from some other code

Reply:
It is all good, it happens to the best of us